lib/dal/src/pkg/export.rs

```diff
@@ -553,7 +518,7 @@ impl PkgExporter {
                 .arity(socket.arity())
                 .ui_hidden(socket.ui_hidden());
 
-            let mut has_custom_func = false;
+            // let mut has_custom_func = false;
```

Comment:
What's going on with this flag?

Reply:
I have no idea! We used to check it before setting the data, but it doesn't make sense to me to only set data if has custom func.

What do you think?

But I didn't want to remove that from the code, since we may want to restore it quickly and without missing information (I'm focusing on making it work to merge this, as the PR has lived for too long and it gets harder and harder to make all parts move together while rebasing on main)